test/func/methodcall/proto/buf.yaml

```diff
@@ -1,9 +1,10 @@
 version: v1
-breaking:
-  use:
-    - FILE
 lint:
-  use:
-    - DEFAULT
-deps:
-  
\ No newline at end of file
+    use:
+        - DEFAULT
+    except:
+        - ENUM_ZERO_VALUE_SUFFIX
```

Comment:
I would really like  to know how this was changed.

Reply:
🫣looks like you emptied it in commit 87240c95bb3a141bd6d0ca996c8f097c7a9c0554